### PR TITLE
pythonPackages.pygraphviz: fix build

### PR DESCRIPTION
**Pull request by rnhmjoj**
###### Motivation for this change
See issue #26738 

###### Things done

- [x] Tested using sandboxing
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Loaded the library and run an [example](https://pygraphviz.github.io/examples.html)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/pygraphviz/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "00ck696rddjnrwfnh1zw87b9xzqfm6sqjy6kqf6kmn1xwsi6f19a";
+  };
+
+  buildInputs = [ doctest-ignore-unicode mock nose ];
+  propagatedBuildInputs = [ graphviz pkgconfig ];
+
+  doCheck = false;
```
Comment (FRidh) [inline pkgs/development/python-modules/pygraphviz/default.nix]: always include a comment motivating why

**Comment by FRidh** [comment]
thanks

**Comment by rnhmjoj** [comment]
@FRidh Can you cherry-pick into 17.03 so we can close #26738?

**Comment by FRidh** [comment]
2fee07561f

**Comment by rnhmjoj** [comment]
Thank you.

**Pull request merged**